make install fails to find kustomization.yaml

<!-- Please use this template while reporting a bug and provide as much info as
possible. Not doing so may result in your bug not being addressed in a timely
manner. Thank you!
-->

**What happened**:
```
make install
kubectl kustomize config/crd/v1alpha2 | kubectl apply -f -
Error: unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory 'gateway-api/config/crd/v1alpha2'
error: no objects passed to apply
make: *** [crd] Error 1
```

**What you expected to happen**:
`make install` to pick up the [`kustomization.yaml`](https://github.com/kubernetes-sigs/gateway-api/blob/master/config/crd/kustomization.yaml) file from the parent directory, `config/crd`, and `make install` to complete successfully.

I'm unsure if this file should be symlinked into the `v1alpha*` directories, distinct copies maintained within each, or if some CLI argument to look for it a level above is missing.

**How to reproduce it (as minimally and precisely as possible)**:
`make install`

**Anything else we need to know?**:
gateway-api `master` branch, 08ffbedace61a5334ad052eac90edf9c40422ae1 
```
kustomize version
{Version:kustomize/v4.4.0 GitCommit:63ec6bdb3d737a7c66901828c5743656c49b60e1 BuildDate:2021-09-27T16:13:36Z GoOs:darwin GoArch:amd64}
```

--- a/Makefile
+++ b/Makefile
@@ -51,7 +51,7 @@ install: crd example
 # Install the CRD's to a pre-existing cluster.
 .PHONY: crd
 crd:
-	kubectl kustomize config/crd/v1alpha2 | kubectl apply -f -
+	kubectl kustomize config/crd | kubectl apply -f -
 
 # Install the example resources to a pre-existing cluster.
 .PHONY: example

